### ISSUE TITLE
Header links get hidden if there is a header image

### ISSUE THREAD

**Issue by emmarichardson** [gjb2048/moodle-theme_foundation#34]
**Is your feature request related to a problem? Please describe.**
I use a background image for my header.   Moodle places certain search bars or links in the header menu (for example, Admin Search Bar and Course Bulk Edit link (I literally only found this functionality today because it was so hard to see with my header background!!  Lol!).  These links/search bars are hard to see if you have a background image.  As I continue to work on becoming fully accessible, this came up as an issue...

**Describe the solution you'd like**
An option to place additional header links somewhere else (below the header?) or give them a white/black background (or just remove the transparency) so that they are visible.

**Describe alternatives you've considered**
I am using css to overwrite the transparent background on the admin search.  Still working on the Bulk Edit link..still searching if there are more secret links that I have never seen before!!

**Additional context**
<img width="1501" alt="Screenshot 2024-04-23 at 2 28 38 PM" src="https://github.com/gjbarnard/moodle-theme_foundation/assets/2079129/7e8db301-f8d1-4091-9a30-221fa2d273fc">

**Versions you are currently using (please complete the following information):**
 - Moodle: [e.g. 4.3.3]
 - Theme: [e.g. 403.1.3]
 - Browser and version [various]

**Comment by gjb2048** [comment]
Example custom SCSS / CSS solution:

```
.header-actions-container {
    background-color: rgba(255,255,255,0.5);
}
```

**Comment by emmarichardson** [comment]
Thanks Gareth - that seems all encompassing after my separate fixes!!

**Comment by gjb2048** [comment]
It isn't perfect Emma but it's a start.

**Comment by emmarichardson** [comment]
My solution was 
![image](https://github.com/gjbarnard/moodle-theme_foundation/assets/2079129/4ecae2ef-1239-4fdd-8462-bef1a32c68eb)

**Comment by gjb2048** [comment]
I see, but those CSS selectors are too generic and can be used elsewhere where they'd cause issues.

**Comment by emmarichardson** [comment]
Yes, I switched to yours - they were working but I also had not looked through every page on the site to see if issues showed up .

**Comment by gjb2048** [comment]
@emmarichardson Is this still an issue please?

**Comment by emmarichardson** [comment]
I think you can close this out.  Thanks.
